python/mxnet/ndarray/numpy/_op.py

```diff
@@ -38,8 +38,7 @@
            'std', 'var', 'indices', 'copysign', 'ravel', 'hanning', 'hamming', 'blackman', 'flip',
            'around', 'hypot', 'rad2deg', 'deg2rad', 'unique', 'lcm', 'tril', 'identity', 'take',
            'ldexp', 'vdot', 'inner', 'outer', 'equal', 'not_equal', 'greater', 'less', 'greater_equal', 'less_equal',
-           'hsplit', 'rot90', 'einsum', 'true_divide']
-
+           'hsplit', 'rot90', 'einsum', 'true_divide', 'column_stack']
```

Comment:
Please move 'column_stack' to after `vstack` in this list.
Also please move the function definition to after `vstack` as well in the file.